### ISSUE TITLE
The sensitivity of the game is always felt differently. What can be done?

### ISSUE THREAD

**Issue by qc0nst** [haxball/haxball-issues#2099]
Whenever I go to play a game, for probably a year now, I have different sensitivities. My extrapolation always stands at the same value - 200, but the game is always different. Every time I restart my computer, restart the browser, or page, I encounter a completely different perception and sensitivity of the game. Despite the fact that my extrapolation is 200, sometimes it can reach such an absurdity that it will feel like 0, that is, various fluctuations will be completely absent. Or on the contrary, these fluctuations will be so powerful that it will seem that the extrapolation is 500. It doesn't depend on the browser, it just happens everywhere for me. I can't play adequately, because one day I have one sensitivity, and the other day it's completely different, just to play the game I need to sit and restart the browser for a certain amount of time to get at least some sensitivity, similar to the extrapolation of 200, and not like 0 and 500. Is it possible to solve this somehow and what could be the reason?

**Comment by kojo953** [comment]
This issue is known since haxball was build in flash. But no one knows why

**Comment by selimozo** [comment]
Yes, idk why it's happening but this is the biggest problem of the game for me
